src/lib_ccx/params.c

```diff
@@ -295,28 +295,28 @@ void print_usage (void)
 	mprint ("  Extracts closed captions and teletext subtitles from video streams.\n");
 	mprint ("    (DVB, .TS, ReplayTV 4000 and 5000, dvr-ms, bttv, Tivo, Dish Network,\n");
 	mprint ("     .mp4, HDHomeRun are known to work).\n\n");
-	mprint ("  Syntax:\n");
+	mprint ("  Usage:\n");
 	mprint ("  ccextractor [options] inputfile1 [inputfile2...] [-o outputfilename]\n");
 	mprint ("\n");
 	mprint ("To see This Help Message: -h or --help\n\n");
 	mprint ("File name related options:\n");
-	mprint ("            inputfile: file(s) to process\n");
-	mprint ("    -o outputfilename: Use -o parameters to define output filename if you don't\n");
-	mprint ("                       like the default ones (same as infile plus _1 or _2 when\n");
-	mprint ("                       needed and file extension, e.g. .srt).\n");
-	mprint ("         -cf filename: Write 'clean' data to a file. Cleans means the ES\n");
-	mprint ("                       without TS or PES headers.\n");
-	mprint ("              -stdout: Write output to stdout (console) instead of file. If\n");
-	mprint ("                       stdout is used, then -o, -o1 and -o2 can't be used. Also\n");
-	mprint ("                       -stdout will redirect all messages to stderr (error).\n");
-	mprint ("           -pesheader: Dump the PES Header to stdout (console). This is\n");
-	mprint ("                       used for debugging purposes to see the contents\n");
-	mprint ("                       of each PES packet header.\n");
-	mprint ("         -debugdvbsub: Write the DVB subtitle debug traces to console.\n");
-	mprint ("      -ignoreptsjumps: Ignore PTS jumps (default).\n");
-	mprint ("         -fixptsjumps: fix pts jumps. Use this parameter if you\n");
-	mprint ("                       experience timeline resets/jumps in the output.\n");
-	mprint ("               -stdin: Reads input from stdin (console) instead of file.\n");
+	mprint ("            inputfile:  file(s) to process\n");
```

Comment:
Why all those extra spaces in the help? Are they needed for the man?
Did you check that every line is still below 80 chars?

Reply:
Atleast 2 spaces are required after every param for help2man to differentiate b/w the parameter and the the decription.